src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -425,7 +436,12 @@ static int _PaPulseAudio_ProcessAudio(PaPulseAudio_Stream *stream,
             size_t tmpSize = pulseaudioOutputBytes;
 
             /* Allocate memory to make it faster to output stuff */
-            pa_stream_begin_write( stream->outputStream, &bufferData, &tmpSize );
+            if( pa_stream_begin_write( stream->outputStream, &bufferData, &tmpSize ) )
+            {
+                PA_DEBUG( ("Portaudio %s: Can't output to stream!\n",
+                __FUNCTION__) )
+                return paNotInitialized;
```

Comment:
Is there a reason to use `paNotInitialized` here but `paInsufficientMemory` in the other case of checking `pa_stream_begin_write`? I think either they should use the same error code, or there should be a comment explaining what's going on.